dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.models;
+
+import uk.ac.ebi.eva.commons.core.models.Region;
+
+import java.math.BigDecimal;
+
+/**
+ * Wrapper for an SS ID, associated RS ID if any, along with its contig and (optionally) chromosome coordinates.
+ */
+public class SubSnpCoreFields {
+
+    private long ssId;
+
+    private Integer rsId;
+
+    private Region contigRegion;
+
+    private Region chromosomeRegion;
+
+    private Orientation snpOrientation;
+
+    private Orientation contigOrientation;
+
+    /**
+     * @param ssId Unique SS ID identifier
+     * @param rsId Unique RS ID identifier, can be null if the SS ID has not been clustered yet
+     * @param snpOrientation Orientation of the SS ID (forward/reverse)
+     * @param contig Contig name
+     * @param contigStart Start coordinate in contig
+     * @param contigEnd End coordinate in contig
+     * @param contigOrientation Orientation of the contig (forward/reverse)
+     * @param chromosome Chromosome name, can be null if the contig is not mapped to a chromosome
+     * @param chromosomeStart Start coordinate in chromosome, null if the contig is not fully mapped to a chromosome
```

Comment:
Are chromosomeStart and chromosomeEnd the coordinates of the variant in the chromosome, or the coordinates of the contig in the chromosome? I guess they are the variant ones, but just to be sure